forte/api/integrals_api.cc

```diff
@@ -77,9 +78,12 @@ void export_ForteIntegrals(py::module& m) {
             "Return the beta-beta 2e-integrals in physicists' notation")
         .def("set_nuclear_repulsion", &ForteIntegrals::set_nuclear_repulsion,
              "Set the nuclear repulsion energy")
+        .def("nuclear_repulsion_energy", &ForteIntegrals::nuclear_repulsion_energy, "Get the nuclear repulsion energy")
+        .def("frozen_core_energy", &ForteIntegrals::frozen_core_energy, "Get the frozen core energy")
         .def("set_scalar", &ForteIntegrals::set_scalar, "Set the scalar energy")
         .def("set_oei", &ForteIntegrals::set_oei_all, "Set the one-electron integrals")
         .def("set_tei", &ForteIntegrals::set_tei_all, "Set the two-electron integrals")
+        .def("build_from_asints", &ForteIntegrals::build_from_asints, "Build the integrals from as_ints.")
```

Comment:
I think the name is a bit confusing.  Is this just setting some integrals using `set_tei`, etc?

Reply:
I agree with York. The key issue is that you are _setting_ not _building_.

Reply:
I change the name to set_ints_from_asints.